docs/ENTITY_RELATIONSHIP_DIAGRAM.md

```diff
@@ -80,8 +76,26 @@ erDiagram
     string log
     datetime createdAt
   }
+  DataProvider {
+    int id
+    string name
+    datetime createdAt
+  }
+  Funder {
+    int id
+    string name
+    datetime createdAt
+  }
+  Source {
+    int id
```

Comment:
`ExternalFieldValue` and `ProposalVersion` (and perhaps `ProposalFieldVersion`) need a reference to this `id`, e.g. `sourceId`.